### DISCUSSION TITLE
Layout shouldn't be persistent in Mobile app and desktop

### DISCUSSION THREAD

**Discussion by rohit-espranza** [dailydotdev/daily#429] [Feature Requests]
I have set the layout in the mobile app to List because there is less space.
But this changes the listings in desktop extension as well, where I want the regular grid style.
Can't we have different layout for different screen.

**Comment by idoshamun** [comment]
Hi @rohit-espranza,

Let me take it back to the team and I'll get back to you :)

**Reply by rohit-espranza** [reply]
Sure thing
